tests/test_builtins.py

```diff
@@ -239,6 +239,37 @@ def test_if_parallel(self):
             assert np.all(a[::-1, 3:6] - np.array(f.data[12:18, 9:12]) == 0)
             assert np.all(a[3:6, ::-1] - np.array(f.data[9:12, 12:18]) == 0)
 
+    @pytest.mark.parametrize('ndim, nbl', [
```

Comment:
I think we should have a test with mpi also?

Reply:
Yes we should, working on it

Reply:
Added MPI test